experimental/framework/tree-react-api/api-report/tree-react-api.beta.api.md

```diff
@@ -5,15 +5,10 @@
 ```ts
 
 // @public
-export interface IReactTreeDataObject<TSchema extends ImplicitFieldSchema> extends ITreeDataObject<TSchema> {
-    readonly TreeViewComponent: (props: TreeViewProps<TSchema>) => React_2.JSX.Element;
-}
-
-// @public
-export interface ITreeDataObject<TSchema extends ImplicitFieldSchema> {
+export interface IReactTreeDataObject<TSchema extends ImplicitFieldSchema> {
     readonly config: TreeViewConfiguration<TSchema>;
-    readonly key: string;
```

Comment:
Since the tree is no longer being inserted into a directory, this `key` property is no longer needed.

Additionally, this base interface was not used outside of the derived `IReactTreeDataObject` interface, so I went ahead and collapsed the two together to simplify things.